lib/shelley/exe/cardano-wallet.hs

```diff
@@ -311,6 +316,12 @@ instance ToText MainLog where
             , "parameters."
             , "Here's (perhaps) some helpful hint:", hint
             ]
+        MsgBlockfrostTokenError tokenFile -> T.unwords
+            [ "File"
+            , T.pack tokenFile
+            , "specified in the --blockfrost-token-file\
+            \ argument doesn't contain a valid Blockfrost API token."
+            ]
```

Comment:
Perhaps it would be nice to have a test against this in `BlockfrostSpec`?

Reply:
Makes sense! I'll add some more tests there.

Reply:
I've added tests for the cases you mentioned above but didn't go as far as testing correspondence between `testnet/mainnet` environment flag and blockfrost token environment as it would require bigger effort to test several flags in combination.